orbit/pkg/table/crowdstrike/falconctl/parser_test.go

```diff
@@ -98,19 +101,19 @@ func TestParseOptions(t *testing.T) {
 				"rfm-reason":      "None",
 				"rfm-reason-code": "0x0",
 				"rfm-state":       "false",
-				"tags":            []string{"kolide-test-1", "kolide-test-2"},
+				"tags":            "kolide-test-1,kolide-test-2",
 			},
 		},
 		{
 			name:  "-rfm-state --rfm-reason --aph --tags --version",
-			input: []byte("aph is not set, rfm-state=false, rfm-reason=None, code=0x0, version = 6.45.14203.0\ntags=kolide-test-1,kolide-test-2,"),
+			input: []byte("aph is not set, rfm-state=false, rfm-reason=None, code=0x0, version = 6.45.14203.0\ntags=kolide-test-1,kolide-test-2, "),
```

Comment:
Does the extra space here test the `if pair == ""` case?  If so consider sticking a comment on there so we don't accidentally remove it.

Reply:
Yep. For the sake of unblocking QA on this I'll add a comment to that effect in another PR.